src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using SFA.DAS.AssessorService.Api.Types.Models.Register;
-using SFA.DAS.AssessorService.Api.Types.Models.UKRLP;
 using SFA.DAS.AssessorService.ApplyTypes;
-using SFA.DAS.AssessorService.ApplyTypes.CharityCommission;
-using SFA.DAS.AssessorService.ApplyTypes.CompaniesHouse;
 using SFA.DAS.AssessorService.ApplyTypes.Roatp;
 using SFA.DAS.AssessorService.ApplyTypes.Roatp.Apply;
-using System.Net.Http.Formatting;
-using SFA.DAS.AdminService.Web.Models;
-using SFA.DAS.AdminService.Web.Infrastructure.RoatpClients.Exceptions;
+using System.Net.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore.Internal;
```

Comment:
I can't see where or why Entity Framework would come into this

Reply:
looks like a misassignment caused by an Any(), However it still worked with the misassignment?  Reset to use System.Linq